ServerSide/yoty/WebApi/PublicDataSchemas/DTO/DealDTO.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) YOTY Corporation and contributors. All rights reserved.
+
+namespace YOTY.Service.WebApi.PublicDataSchemas
+{
+    using System.Collections.Generic;
+
+    public class DealDTO
```

Comment:
lets remove this for now as you said

Reply:
I'm keeping it but remove it from our API layer and managers layer